test/test_backend/db/tst_config.py

```diff
@@ -0,0 +1,48 @@
+"""Tests
+
+How to run:
+    python -m unittest discover
+
+TODO: When done: Tst -> Test
+
+todo: would be nice to have a test to make sure nothing in STANDALONE_TABLES doesn't exist in DB, for cleanliness
+"""
+import os
+import sys
+from pathlib import Path
+
+# noinspection DuplicatedCode
+THIS_DIR = Path(os.path.dirname(__file__))
+PROJECT_ROOT = THIS_DIR.parent.parent.parent
+sys.path.insert(0, str(PROJECT_ROOT))
+
+from backend.db.config import DERIVED_TABLE_DEPENDENCY_MAP, STANDALONE_TABLES
+from backend.db.utils import show_tables
+
+
+class TestBackendDbConfig:
```

Comment:
## `TestBackendDbConfig`
@Sigfried I added a new, important test to make sure that our database table/view dependency configuration is up to date.

I noticed there were some new tables in the DB, but they were not in this configuration. It's important that we remember to always do the following things:
1. If there's a new derived table/view, it need to be added to `DERIVED_TABLE_DEPENDENCY_MAP`
2. If there's a new table/view that is standalone and not derived, and no table/view is derived from it, it needs to be added to `STANDALONE_TABLES`
3. If a table's DDL has changed / it has changed so that it is now dependent on different tables from before, this must be updated to reflect in `DERIVED_TABLE_DEPENDENCY_MAP`

For case (1), I ran the test and found these two new tables:
- `concept_graph_with_rel`: I don't see DDL for it or references in the codebase. @Sigfried just confirming this is an ad hoc and standalone table?
- `cset_term_usage_rec_counts`: I see this is a new table that depends on other tables. I will add update the config. Also FYI this needs to be put in its own DDL.

My test also strangely found this table, but I don't see it in the `n3c` schema! @Sigfried What's up with this one?
- `api_call_group_id_seq`

For case (2), I updated `STANDALONE_TABLES` to reflect:
- 'api_call_group_id_seq', - @Sigfried Is this correct. Standalone? No derived / anything dependent on?
- 'junk',
- 'relationship',
- 'researcher'

For case (3), this one worries me a little. There's not an easy way to make sure that `DERIVED_TABLE_DEPENDENCY_MAP` remains updated. I suppose one really complicated thing would be to write a test that compares the current DDL against the state of the DDL at the time the test last ran successfully, and do some elaborate text parsing to determine if there were dependency changes. If I can do this, then `DERIVED_TABLE_DEPENDENCY_MAP` could also theoretically be created automatically, and we could have an action to update that once in a while, perhaps bundled in with the refresh. Would be really awesome. But it sounds very strange and error-prone.
I did notice that `all_csets` dependencies are not updated. One of the new tables is a dependency. I'll update the config. But I wonder if you can tell me if anything else has changed recently off top of your head.

Reply:
Moved this to new PR